index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Microverse Partnership: Ademola-Agasha</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <div class="container">
+        <div>
```

Comment:
- [ ] **[OPTIONAL]**

- [ ] Hey, great job on implementing the changes required by the previous reviewer, however, you left this one undone but I will overlook and approve it because I think you probably forgot to update it. 
**NOTE** Always try to confirm all changes to be sure you're not leaving anyone undone.